Desktop/Prework/Allegro/allegro/src/components/ListItem/ListItem.module.scss

```diff
@@ -0,0 +1,76 @@
+.wrapper{
+    width: 11rem;
+    height: 17rem;
+    position: relative;
+    background: #FFFFFF;
+    border: 2px solid rgba(0, 0, 0, 0.9);
+    box-shadow: inset 0px 4px 4px rgba(0, 0, 0, 0.25);
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column; 
+    align-items: center;
+    padding-top: 3.5rem;
+    justify-self: center;
```

Comment:
Another good practice is to group similar css properties with eachother. What this means is that for example `width`, `height`, `padding`, `margin` should be next to eachother; `color`, `background-color`, `border` should be next to eachother; flex related properties too. This way it is easier to read.